packages/crew_launcher.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Crew_launcher < Package
+  description 'Add Chromebrew app to launcher'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version '1'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'https://github.com/supechicken/crew-launcher.git'
+  git_hashtag 'main'
```

Comment:
It would be better to tag a version so bug fixes can happen on future releases.  Generally, it's not a good best practice to use the main (or master) branch since it can change over time as new commits are added.

Reply:
Pointed to commit temporarily, I will create a version tag when this pr is ready

Reply:
Can we change the status to Draft until ready?

Reply:
Done